### PR TITLE
Added two theorems about Čech-complete spaces

### PR DESCRIPTION
**Pull request by david20000813**
One additional comment here: should we delete T135, Completely metrizable => Baire, since it now follows from T133, Completely metrizable => Čech-complete, and the proposed T623, Čech-complete => Baire?

See #930.

### CODE REVIEW

**Comment by StevenClontz** [comment]
> One additional comment here: should we delete T135, Completely metrizable => Baire, since it now follows from T133, Completely metrizable => Čech-complete, and the proposed T623, Čech-complete => Baire?

Often in this case we will edit T135 rather than introduce a new T623.

**Comment by prabau** [comment]
It's more than just "asserted" in the referenced theorems.  Better say `See Theorem ...`.

For T623: the $k_3$-space property is the least useful of the three variants of $k$-space in general.  Better use $k_1$-space (P140).  It's all the same anyway for Hausdorff spaces.

File: theorems/T000623.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+if:
+  P000063: true
+then:
+  P000142: true
+refs:
+  - zb: "0684.54001"
+    name: General Topology (Engelking, 1989)
+---
+
+Asserted in Theorem 3.9.5 of {{zb:0684.54001}}.
```
Comment (david20000813) [inline theorems/T000623.md:12]:
```suggestion
See Theorem 3.9.5 of {{zb:0684.54001}}.
```

**Comment by david20000813** [comment]
@prabau The reason I used $k_3$ is because it is the strongest among all $k$-space properties. In particular, it implies $k_2$ then $k_1$. Also why I chose $k_1$ in #944, where asserting not $k_1$ is the strongest available property. Though, of course, we're talking about Hausdorff spaces here, so it really doesn't matter. It just feels better to assert what formally is the strongest possible conclusion.

**Comment by prabau** [comment]
Yeah, I understand.  But there are equivalent in this case.  And practically nobody uses k3-space in the literature.  We introduced it in pi-base to better distinguish the various variants in the definitions.  k1-space is easier to check than the rest in this case.  So I will really insist here.

File: theorems/T000623.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+uid: T000623
+if:
+  P000063: true
+then:
+  P000142: true
```
Comment (david20000813) [inline theorems/T000623.md:6]:
```suggestion
  P000140: true
```

**Comment by david20000813** [comment]
@prabau In that case, sure, I've changed the theorem to imply $k_1$ instead.

**Comment by prabau** [comment]
One additional argument in favor of k1 in this case: the proof in Engelking shows k1 and not k3.  Again, they are all equivalent, but k1 is the more natural version when working with Hausdorff spaces.  k3 is more a strengthening of k2 used to illustrate the reason for the k2 definition, via the shortcomings of k3.

**Review by prabau — APPROVED** [review]

**Pull request merged**